app/resources/api/v2/shared_behaviour/labware.rb

```diff
@@ -7,62 +7,75 @@
 # on its effective subclasses (Liskov Substitution Principle) this is especially
 # true for relationships, as it means we support pre-loading those relationships
 # on mixed endpoints
-module Api::V2::SharedBehaviour::Labware
-  extend ActiveSupport::Concern
+module Api
+  module V2
+    module SharedBehaviour
+      module Labware
+        extend ActiveSupport::Concern
 
-  included do
-    # Associations:
-    has_one :purpose, readonly: true, foreign_key: :plate_purpose_id, class_name: 'Purpose'
-    has_one :custom_metadatum_collection, foreign_key_on: :related
+        included do
+          # Associations:
+          has_one :purpose, readonly: true, foreign_key: :plate_purpose_id, class_name: 'Purpose'
+          has_one :custom_metadatum_collection, foreign_key_on: :related
 
-    has_many :samples, readonly: true
-    has_many :studies, readonly: true
-    has_many :projects, readonly: true
-    has_many :comments, readonly: true
+          has_many :samples, readonly: true
```

Comment:
Were these blank lines added by the linters? 🤔

Reply:
Nope. You've changed the module structure. Got it.

Reply:
The linter said the module had too many lines so I split it in two